book/src/external-provisioner.md

```diff
@@ -23,7 +23,13 @@ Once a new volume is successfully provisioned, the sidecar container creates a K
 
 The deletion of a `PersistentVolumeClaim` object bound to a `PersistentVolume` corresponding to this driver with a `delete` reclaim policy causes the sidecar container to trigger a `DeleteVolume` operation against the specified CSI endpoint to delete the volume. Once the volume is successfully deleted, the sidecar container also deletes the `PersistentVolume` object representing the volume.
 
-The CSI `external-provisioner` also supports the `Snapshot` DataSource. If a `Snapshot` CRD is specified as a data source on a PVC object, the sidecar container fetches the information about the snapshot by fetching the `SnapshotContent` object and populates the data source field in the resulting `CreateVolume` call to indicate to the storage system that the new volume should be populated using the specified snapshot.
+### DataSources 
```

Comment:
Maybe start off with one sentence like "The external-provisioner can provision a volume that is pre-populated from a data source"

Reply:
I broke this up and hopefully it flows better.